dropwizard-servers/src/main/java/com/palantir/remoting1/servers/dropwizard/DropwizardServers.java

```diff
@@ -44,5 +41,6 @@ public void accept(ExceptionMapper<? extends Throwable> mapper) {
                     }
                 });
         environment.jersey().register(new OptionalAsNoContentMessageBodyWriter());
+        environment.jersey().register(new TraceEnrichingFilter());
```

Comment:
we should convert `ExceptionMappers` to `JerseyComponents` and probably change `visitExceptionMappers` to return a `Collection<?>` of those components to register. In most cases, we might want to just return the class instance.